cicd/api-test/deploy-service-api-test.postman_collection.json

```diff
@@ -0,0 +1,138 @@
+                            "    pm.response.to.have.status(200);",
+                            "});"
+                        ],
+                        "type": "text/javascript"
+                    }
+                }
+            ],
+            "request": {
+                "method": "GET",
+                "header": [
+                    {
+                        "key": "Accept",
+                        "value": "application/json"
+                    }
+                ],
+                "url": {
+                    "raw": "{{baseUrl}}v1/deploys/:id",
+                    "host": [
+                        "{{baseUrl}}v1"
+                    ],
+                    "path": [
+                        "deploys",
+                        ":id"
+                    ],
+                    "variable": [
+                        {
+                            "key": "id",
+                            "value": "{{deployId}}",
+                            "description": "(Required) Deploy id"
+                        }
+                    ]
+                },
+                "description": "Returns a deploy object given a deploy id"
+            },
+            "response": [
+                {
+                    "name": "successful operation",
+                    "originalRequest": {
+                        "method": "GET",
+                        "header": [],
+                        "url": {
+                            "raw": "/v1/deploys/:id",
+                            "path": [
+                                "v1",
+                                "deploys",
+                                ":id"
+                            ],
+                            "variable": [
+                                {
+                                    "key": "id",
+                                    "value": "<string>",
+                                    "description": "(Required) Deploy id"
+                                }
+                            ]
+                        }
+                    },
+                    "status": "OK",
+                    "code": 200,
+                    "_postman_previewlanguage": "json",
+                    "header": [
+                        {
+                            "key": "Content-Type",
+                            "value": "application/json"
+                        }
+                    ],
+                    "cookie": [],
+                    "body": "{\n  \"alias\": \"<string>\",\n  \"state\": \"<string>\",\n  \"operator\": \"<string>\",\n  \"description\": \"<string>\",\n  \"total\": \"<integer>\",\n  \"id\": \"<string>\",\n  \"envId\": \"<string>\",\n  \"buildId\": \"<string>\",\n  \"type\": \"<string>\",\n  \"startDate\": \"<long>\",\n  \"lastUpdateDate\": \"<long>\",\n  \"successTotal\": \"<integer>\",\n  \"failTotal\": \"<integer>\",\n  \"successDate\": \"<long>\",\n  \"acceptanceStatus\": \"<string>\",\n  \"fromDeployId\": \"<string>\"\n}"
```

Comment:
I think this is a generated serialized "json in json" escape. I think word wrap in the editor should be good enough, WDYT @ntascii ?

Reply:
I hope in the long term we can find a different way to bring this json in, having nested json is not good for readability or hand-editing (outside postman). I imagine there must be some way to avoid this, like referring to a file.  I think it's probably better to look at as we add more tests.

Reply:
@robbintt yeah I think it works. Hope there is a better way to create a new test case. Having this nested json is pretty error prone and will be hard to debug if there is a test failed.